dev/push-setup/Gemfile

```diff
@@ -0,0 +1,8 @@
+source 'https://rubygems.org'
+
+gem 'berkshelf'
+gem 'test-kitchen'
+gem 'kitchen-vagrant'
+gem 'kitchen-nodes'
+gem 'chef-zero'
+gem 'chef'
```

Comment:
Yeah actually only `kitchen-nodes` should be needed (and `knife-push` as discussed above).